src/applications/claims-status/tests/components/WhatWeAreDoing.unit.spec.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+import { $ } from '@department-of-veterans-affairs/platform-forms-system/ui';
+
+import WhatWeAreDoing from '../../components/WhatWeAreDoing';
+
+const statusMap = {
```

Comment:
We should try and move this code out into someplace like `utils/helpers.js` or something since this code also exists in the `ClaimCard` code

Reply:
Updated!